consul/operations.go

```diff
@@ -0,0 +1,84 @@
+package consul
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/layer5io/meshery-adapter-library/adapter"
+	"github.com/layer5io/meshery-consul/internal/config"
+)
+
+func (h *Handler) ApplyOperation(ctx context.Context, request adapter.OperationRequest) error {
+	operations := make(adapter.Operations)
+	err := h.Config.Operations(&operations)
+	if err != nil {
+		return err
+	}
+
+	status := "deploying"
```

Comment:
Declare the status’ as constants as well (maybe enemy’s)

Reply:
Agreed, that was left for later, but I should have created an issue for it. Status constants (or enums) should be declared in meshery-adapter-library. I'll create an issue, possibly define them here now so that they can be moved to meshery-adapter-library later.